tools/syz-covermerger/syz_covermerger.go

```diff
@@ -96,13 +97,13 @@ func do() error {
 			return fmt.Errorf("dash.CreateUploadURL: %w", err)
 		}
 	}
-	if url != ""{
+	if url != "" {
```

Comment:
This chunk belongs to the previous patch. Either move it, or squash the two patches together.

Reply:
Thanks. Merged them as a cheaper alternative.